tests/nn/pipe/test_parity.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Testing Pipe Module
+"""
+
+import contextlib
+import copy
+
+import numpy as np
+import pytest
+import torch
+
+from fairscale.nn import Pipe
+from fairscale.utils.testing import skip_if_single_gpu
+
+
+def _get_model(num_inputs=2, num_hidden=20, num_outputs=2):
+    num_layers = torch.cuda.device_count() - 2
```

Comment:
Will this always be 0? I don't remember if we run our tests on 4 GPU or only 2  on circleCI.

Reply:
we are using 4 GPUs